test/torch_np/numpy_tests/core/test_numeric.py

```diff
@@ -2991,14 +2991,14 @@ def test_broadcast_error_kwargs(self):
         # gh-13455
         arrs = [np.empty((5, 6, 7))]
         mit = np.broadcast(*arrs)
-        mit2 = np.broadcast(*arrs, **{})
+        mit2 = np.broadcast(*arrs, **{})  # noqa: PIE804
```

Comment:
Does it really need kwargs?
```suggestion
        mit2 = np.broadcast(*arrs)
```

Reply:
if we remove it, then it's the same as mit 1 which seems like we are reducing coverage?